packages/main/test/specs/MultiInput.spec.js

```diff
@@ -213,21 +213,38 @@ describe("Keyboard handling", () => {
 		const innerInput =  await input.shadow$("input");
 		const firstToken = await browser.$("#basic-overflow ui5-token:first-child");
 		const lastToken = await browser.$("#basic-overflow ui5-token:last-child");
+		let caretPosition;
 
 		await innerInput.click();
 		await innerInput.keys("ArrowLeft");
 		assert.strictEqual(await lastToken.getProperty("focused"), true, "The last token is focused");
 
 		await innerInput.keys("Home");
-		assert.strictEqual(await firstToken.getProperty("focused"), true, "The last token is focused");
+		assert.strictEqual(await firstToken.getProperty("focused"), true, "The first token is focused");
 
 		await innerInput.keys("End");
 		assert.strictEqual(await lastToken.getProperty("focused"), true, "The last token is focused");
 
 
 		await innerInput.keys("ArrowRight");
-		assert.strictEqual(await firstToken.getProperty("focused"), false, "The last token is not focused anymore");
+		assert.strictEqual(await firstToken.getProperty("focused"), false, "The first token is not focused anymore");
 		assert.strictEqual(await lastToken.getProperty("focused"), false, "The last token is not focused anymore");
+
+		await innerInput.keys("Backspace");
```

Comment:
Please, add a fully functional sample, where "token-delete" event is used and a token can be deleted on backspace. We should have such in order to be able to test that functionality. Then using that sample, please add a test whether a token is deleted on BACKSPACE

Reply:
Done